release/docker/config/opensearch/opensearch-docker-entrypoint.sh

```diff
@@ -71,6 +74,25 @@ done < <(env)
 # will run in.
 export OPENSEARCH_JAVA_OPTS="-Dopensearch.cgroups.hierarchy.override=/ $OPENSEARCH_JAVA_OPTS"
 
+##Security Plugin
+SECURITY_PLUGIN="opensearch-security"
+if [ -d "$OPENSEARCH_HOME/plugins/$SECURITY_PLUGIN" ]; then
+    if [ "$DISABLE_INSTALL_DEMO_CONFIG" = "true" ]; then
+        echo "Disable Install Demo Config for OpenSearch Security Plugin"
```

Comment:
Please say `Disabling Install Demo Config for OpenSearch Security Plugin` and 

`Enabling Install Demo Config for OpenSearch Security Plugin`

Reply:
fixed.